src/app/shared/entry.ts

```diff
@@ -259,11 +276,51 @@ export abstract class Entry implements OnInit, OnDestroy {
    * @param {Tag|WorkflowVersion} b - version b
    * @returns {number} - indicates order
    */
-  entryVersionSorting(a: Tag | WorkflowVersion, b: Tag | WorkflowVersion): number {
+  entryVersionSorting(a: Tag | WorkflowVersion, b: Tag | WorkflowVersion, isTool: boolean): number {
+    // let entryADate: any;
```

Comment:
In general, unless you expect to restore this, you shouldn't leave this commented out code